File: Training/OOPFundamentals/Java/Exercises/after/08 Packages and Interfaces/src/com/daugherty/gadgets/CalculatorWatch.java
package com.daugherty.gadgets;

import com.daugherty.math.ICalculator;
import com.daugherty.time.IWatch;

public class CalculatorWatch implements ICalculator, IWatch {


    @Override
    public int calculateAdd(int numberOne, int numberTwo) {
        return numberOne + numberTwo;
    }

    @Override
    public long getUNIXEpochTime() {
        return System.currentTimeMillis();
    }
}

File: Training/OOPFundamentals/C#/Exercises/after/PackagesAndInterfaces/Gadgets/CalculatorWatch.cs
using PackagesAndInterfaces.Math;
using PackagesAndInterfaces.Time;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PackagesAndInterfaces.Gadgets
{
    public class CalculatorWatch : IWatch, ICalculator
    {
       
        public long GetUNIXEpochTime()
        {
            long time = (long)(DateTime.UtcNow - new DateTime(1970, 1, 1)).TotalSeconds;
            return time;
        }
        public int CalculateAdd(int numberOne, int numberTwo)
        {
            return numberOne + numberTwo;
        }
    }
}
